### PR TITLE
Gjengroing first commit

### PR DESCRIPTION
**Pull request by idammie**

### CODE REVIEW

**Comment by anders-kolstad** [comment]
Hi @idammie , what is the status for the PR? Can I start reviewing it?

**Comment by idammie** [comment]
> Hi @idammie , what is the status for the PR? Can I start reviewing it?

@anders-kolstad you can start reviewing the PR.

**Comment by anders-kolstad** [comment]
@idammie , can you please delete some of the unneeded files from the PR? 

- [ ] You have four versions of the qmd file. Assuming NO_GJEN_004 is the correct one, can you please delete the other ones, and rename the correct one NO_GJEN_004. 
- [ ] And same with the htmls. 
- [ ] Please delete 'ecosystemCondition_gjengroing.Rmd' 

You can also make me admin in your repo if you want me to do it for you.

**Comment by anders-kolstad** [comment]
> @idammie , can you please delete some of the unneeded files from the PR?
> 
> * [ ]  You have four versions of the qmd file. Assuming NO_GJEN_004 is the correct one, can you please delete the other ones, and rename the correct one NO_GJEN_004.
> * [ ]  And same with the htmls.
> * [ ]  Please delete 'ecosystemCondition_gjengroing.Rmd'
> 
> You can also make me admin in your repo if you want me to do it for you.

Actuall, hang on a bit @idammie. I will fork your fork and make some edits there. I will assume NO_GJEN_004 is the one to keep.

File: indicators/NO_GJEN/R/NO_GJEN_004.qmd [hunk 1]
```diff
@@ -0,0 +1,1467 @@
...
+knitr::include_graphics("../img/gjengroing_schematic.jpg")
+```
+
+The NiN polygons including "Våtmark", "Naturlig åpne områder under skoggrensa" and "Semi-naturlig mark" with good ecological condition are used to define a reference gjengroing state. We use the aggregated "Tilstand" variable assigned to each NiN polygon. The 50th percentile of LiDAR-derived vegetation heights within these polygons is used to define the upper limit (ie. 1) of the scaled indicator value. We cannot define local reference values based on proximity, because the NiN polygons are spatially biased and not close to all population polygons. Therefore we calculate regional reference values using regions and bioclimatic zones as stratification. We calculate the mean reference value for each unique combination of region-bioclimatic zone. When calculating the index for each population polygon, the reference value is inherited from the region-bioclimatic zone it falls within.
```
Comment (anders-kolstad) [inline indicators/NO_GJEN/R/NO_GJEN_004.qmd:203]: I think it is best to use the variation in the estimated indicator values to describe the regional uncertainty. This is more aligned with the ecosystem condition framework, bith the Norwegian Fagsystem and the SEEA EA

File: indicators/NO_GJEN/R/NO_GJEN_004.qmd [hunk 2]
```diff
@@ -0,0 +1,1467 @@
...
+
+To define the ecological condition of the population polygon, we measure 50th percentile of LiDAR-derived vegetation height. Population polygons should be approximately compatible with the NiN ecosystem types, and we have thus used three different data sources for the three different ecosystem types. For våtmark, we use types in nasjonalt grunnkart defined as Klasse 7 Våtmark. For åpen, we have used the GRUK dataset. For semi-natural, we have used the ASO dataset.
+
+The vegetation height percentiles are then scaled to between 0 and 1 using a sigmoid transformation [Oliver at al. (2021)](https://www.sciencedirect.com/science/article/pii/S1470160X21000066).
```
Comment (anders-kolstad) [inline indicators/NO_GJEN/R/NO_GJEN_004.qmd:209]: Please add a little more reasoning why so, and why not say an expoential transformation as suggested here: https://github.com/NINAnor/ecosystemCondition/issues/156

File: indicators/NO_GJEN/R/NO_GJEN_004.qmd [hunk 3]
```diff
@@ -0,0 +1,1467 @@
...
+  indicator_LowHigh <- (variable - refLow)/(refHigh - refLow)
+  indicator_LowHigh[indicator_LowHigh[] < 0] <- 0
+  indicator_LowHigh[indicator_LowHigh[] > 1] <- 1
+  indicator_sigmoid <- 100.68*(1-exp(-5*(indicator_LowHigh)^2.5))/100
+  return (round(indicator_sigmoid, 4))
+}
+
+```
+
+### 9.1 Scaled indicator values at polygon level
```
Comment (anders-kolstad) [inline indicators/NO_GJEN/R/NO_GJEN_004.qmd:889]: I added two more figures in this section. Please have a look and see if they make sense.

**Review by anders-kolstad — CHANGES_REQUESTED** [review]
Hi. Thank you @idammie. I have finished reviewing the PR. Instead of making comments for on stuff to fix, I tried to suggest as many things as possible directly in a [PR to your fork.](https://github.com/idammie/ecRxiv_gjengroing/pull/1). Please review that PR and merge it to idammie:gjengroing if you  find it OK. This will automatically update your PR to ecRxiv. See for example the diff file [here](https://github.com/idammie/ecRxiv_gjengroing/pull/1/files#diff-5bb9d3370286a69798e985a7df71ed105fba9e8c5947dae9bea9430006db0419). 

I also added some suggestions for improvement beyond what I could do in the review, esp:

1. using indicator values to describe the uncertainty
2. using *rask suksesjon* instead of *tilstand*
3. validating by plotting indicator values against *rask susesjon* across all NiN polygons.

Se details for these issues inside the review.
[Here ](http://htmlpreview.github.io/?https://github.com/NINAnor/ecRxiv_gjengroing_review/blob/review_ALK/indicators/NO_GJEN/R/NO_GJEN_001.html)you can also see what my reviewed version looks like when rendered.

Thanks for submitting and dont hessitat to ask if anything is unclear and I can try to explain better. 

Cheers.

**Pull request merged**